### PR TITLE
Deployment parsing

### PR DESCRIPTION
**Pull request by Sebastian-Fransson**

### CODE REVIEW

**Review by timjon — DISMISSED** [review]
Looks good

**Review by timjon — DISMISSED** [review]

**Review by kosara-g — DISMISSED** [review]
Works fine.

File: Application/src/controller/Import.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import java.util.List;
 
 /**
- * @version 1.0
```
Comment (IsabelleTornqvist) [inline Application/src/controller/Import.java:18]: Contributor tag

Comment (Sebastian-Fransson) [reply Application/src/controller/Import.java:18]: Fixed

**Review by kosara-g — APPROVED** [review]
Still fine.

**Pull request merged**